docs/DocumentationStyleGuide/style-tone.rst

```diff
@@ -0,0 +1,90 @@
+Style & Tone
+===========
+
+We aim to achieve a simple, easy-to-understand context for our technical
+and non-technical documentation. It aims to overlook the too-formal or
+robotic tone and give way to a friendly, inclusive and respectful tone
+of communication. It should call to action in a casual and approachable
+tone and should not sound pushy or dry.
+
+As a Technical Writer, if you are using an entertaining tone to help the
+readers better understand, do not go over the humour part. We generally
+assume that not every humour piece is universal and can be interpreted
+differently across different regions.
+
+A few guidelines are employed to understand the style and tone. These
+are entirely flexible and are not universal for every context that we
+share amongst ourselves.
+
+Approaches to be considered
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  Use Active voice wherever possible.
+-  The style and tone should be friendly and come off as natural.
+-  Making use of transaction words is quite preferred.
+-  Simplify the documentation hierarchy for the readers to easily
```

Comment:
Can you add an example of a simple document hierarchy?

Reply:
It would be added in the content strategy and we can later link it here.